frontend/src/entry/model/mirroredModels/ExportSettings.tsx

```diff
@@ -56,11 +56,9 @@ export default function ExportSettings({ model }: ExportSettingsProps) {
   return (
     <>
       <ExportModelAgreement modelId={model.id} />
-      <Accordion sx={{ borderTop: 'none' }}>
+      <Accordion sx={{ borderTop: 'none' }} slotProps={{ heading: { component: 'h3' } }}>
```

Comment:
I'm not sure we want this change. We specifically want the text inside the heading to be a `h3`. Consider reverting this change unless the upgrade guide specifically mentions this is the correct way to do accessibility.